packages/gatsby-source-shopify/__tests__/fixtures/bulk-results/collections.jsonl

```diff
@@ -1 +1 @@
-{"description":"Test Collection Description","descriptionHtml":"Test Collection Description","feedback":{"details":[],"summary":"This collection couldn’t be published to Google"},"handle":"test-collection","id":"gid:\/\/shopify\/Collection\/278407151822","image":null,"legacyResourceId":"278407151822","productsCount":2,"ruleSet":{"appliedDisjunctively":false,"rules":[{"column":"IS_PRICE_REDUCED","condition":"","relation":"IS_NOT_SET"}]},"seo":{"description":null,"title":null},"sortOrder":"BEST_SELLING","storefrontId":"Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzI3ODQwNzE1MTgyMg==","templateSuffix":"","title":"Test Collection","updatedAt":"2022-03-14T22:39:00Z"}
+{"description":"Test Collection Description","descriptionHtml":"Test Collection Description","feedback":{"details":[],"summary":"This collection couldn’t be published to Google"},"handle":"test-collection","id":"gid:\/\/shopify\/Collection\/278407151822","image":null,"legacyResourceId":"278407151822","productsCount":{count: 2, "precision": "EXACT"},"ruleSet":{"appliedDisjunctively":false,"rules":[{"column":"IS_PRICE_REDUCED","condition":"","relation":"IS_NOT_SET"}]},"seo":{"description":null,"title":null},"sortOrder":"BEST_SELLING","storefrontId":"Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzI3ODQwNzE1MTgyMg==","templateSuffix":"","title":"Test Collection","updatedAt":"2022-03-14T22:39:00Z"}
```

Comment:
Those fixtures were modified by hand? (seeing missing `"` around `count` field)

I'll try to see if I can properly regenerate them to get actual data (if I can find the store that was used to generate those fixtures)

Reply:
Welp, definitely won't have access to store because those were not generated by gatsby maintainers so editing by hand will have to do. But this should be validated their correctness by doing some comparisons

Reply:
I did update syntaxes (and corresponding snapshots) just to get over unit tests and see if some later tests fail. It does need some work, because I don't think with changes as-is it will work.

In particular referencing Shopify types like `Count` or `Handle` in gatsby schema types won't work - we will need to create those types ourselves first so we could use them in gatsby's data layer.